pom.xml

```diff
@@ -257,6 +257,22 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
+      <id>central_without_mirror</id>
+      <!--
+        This is used as a fallback when a mirror to `central` fail.
+        For example, when we use Google Maven Central in GitHub Action as a mirror of `central`,
+        this will be used when Google Maven Central is out of sync due to its late sync cycle.
+      -->
```

Comment:
Oh, I mistakenly thought the maven automatically would do the fallback in the mirror failure case.

How about leaving some comments, too, about this behaivour in the script of GitHub Actions?
https://github.com/apache/spark/blob/19a10597a8eaa5e7688d2e472b62a1feeb91c3d5/.github/workflows/master.yml#L69

Reply:
Sure, @maropu .

Reply:
Thanks!